source/index.txt

```diff
@@ -101,6 +102,13 @@ What's New
 For a list of new features and changes in each version, see the
 :doc:`What's New </whats-new>` section.
 
+Upgrade Driver Versions
+-----------------------
+
+Learn what changes you may need to make to your application to upgrade
+versions or from the Legacy API to the current one in the
```

Comment:
Issue: I believe "Legacy" in "legacy API" is inconsistently capitalized. I'm not sure which is right, but elsewhere in the PR it is lower case.

```suggestion
versions or from the legacy API to the current one in the
```

Reply:
Good catch, thought I had updated all of the instances. Initially, I started with capital "L", but noticed the FAQ entries used lowercase, and switched to the latter.